js/about/preferences.js

```diff
@@ -1248,16 +1219,17 @@ class PaymentsTab extends ImmutableComponent {
               <td>
                 <SettingsList>
                   <SettingItem>
-                    <select className='form-control fundsSelectBox'
+                    <FormDropdown
```

Comment:
no; I manually tested and it doesn't do anything (it still stays ~10px). Also, the "Add funds" button has a similar padding. If you remove padding from "Add funds" button (ex: set to 0), it looks bad